Gems/EMotionFX/Code/MCore/Source/StringIdPool.cpp

```diff
@@ -10,8 +10,12 @@
 #include "StringIdPool.h"
 #include "LogManager.h"
 
+#include <AzCore/Serialization/SerializeContext.h>
+
 namespace MCore
 {
+    AZ_TYPE_INFO_SPECIALIZE_WITH_NAME_IMPL(StringIdPoolIndex, "StringIdPoolIndex", "{C374F051-8323-49DB-A1BD-C6B6CF0333C0}");
```

Comment:
Does this need to have the name "MCore::StringIdPoolIndex" or is it never serialized out to storage?

Reply:
This is never serialized to storage, so the including the name of "MCore::" isn't needed.